### DISCUSSION TITLE
Add Vue documentation

### DISCUSSION THREAD

**Discussion by chrisgliddon** [DRincs-Productions/pixi-vn#258] [Wiki chat (giscus)]
Currently there is no documentation about how to use Vue with this. Any docs would be helpful!

**Comment by BlackRam-oss** [comment]
The reason why it's not there is that I've never used Vue.js, I only know how to use React for UI at the moment.
It should work similar to React: https://pixi-vn.web.app/start/interface-react.html

Are you a Vue.js programmer? If so, could you help me create a documentation to add pix-vn to a Vue.js project (it shouldn't take much)? and create a template for Vue.js?

**Comment by BlackRam-oss** [comment]
I have added documentation explaining how to use pixi-vn in a Vue application. It will be posted on the site tomorrow
